packages/radix-vue/src/HoverCard/HoverCardRoot.vue

```diff
@@ -66,11 +67,15 @@ const closeTimerRef = ref(0)
 const hasSelectionRef = ref(false)
 const isPointerDownOnContentRef = ref(false)
 const isPointerInTransit = ref(false)
+const isHoveringRef = ref(false)
 const triggerElement = ref<HTMLElement>()
 
 function handleOpen() {
   clearTimeout(closeTimerRef.value)
-  openTimerRef.value = window.setTimeout(() => open.value = true, openDelay.value)
+  openTimerRef.value = window.setTimeout(() => {
+    if (isHoveringRef.value)
+      open.value = true
```

Comment:
Thanks for the PR @ianfireman . Adding the  `isHoveringRef` causes the `focus` from Tab doesn't open the HoverCard anymore.

The tab focus behavior should be as such:

https://github.com/radix-vue/radix-vue/assets/59365435/2e91dc39-baf9-4a61-9490-7bbbb6f3682c

Reply:
Hmm, gonna fix it

Reply:
@zernonia Now the behavior should be the same, thanks for the feedback!